platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/layers/Property.java

```diff
@@ -11,7 +11,9 @@
  */
 public abstract class Property<T> {
 
-    //visibility
+    /**
+     * Layer visibility
+     */
     public static final String VISIBLE = "visible";
```

Comment:
@incanus If you document in this way, it is only applied to the first constant `VISIBLE`. The `//` I used don't show up at all in javadoc. I use those more as you'd use `#pragma mark`, to organize the source, not to document. If you apply javadoc, you need to do it on all constants I'm afraid.